### ISSUE TITLE
Conference organiser should be able to assign abstracts to reviewers

### ISSUE THREAD

**Issue by msmichellegar** [OxfordAbstracts/application#14]
As a conference organiser, I should be able to assign abstracts to certain users to review. 

---

Current screen:

![screen shot 2016-02-12 at 2 55 17 pm](https://cloud.githubusercontent.com/assets/10683087/13010235/ac5b20e0-d198-11e5-9308-d0d8e482fad6.png)

**Comment by iteles** [comment]
@msmichellegar I agree with you completely, it seems unrealistic to do this via serial numbers. It would be good to sketch out a couple of different options here and test which is more natural to the users.
